plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/FeatureUtils.java

```diff
@@ -1303,16 +1303,23 @@ public static final void setParent(IFeature feature, IFeature newParent) {
 	public static void setRelevantConstraints(IFeature bone) {
 		requireNonNull(bone);
 
-		final List<IConstraint> constraintList = new LinkedList<>();
+		final List<Constraint> constraintList = new LinkedList<Constraint>();
 		for (final IConstraint constraint : bone.getFeatureModel().getConstraints()) {
 			for (final IFeature f : constraint.getContainedFeatures()) {
 				if (f.getName().equals(bone.getName())) {
-					constraintList.add(constraint.clone(bone.getFeatureModel()));
+					constraintList.add((Constraint) constraint.clone(bone.getFeatureModel()));
 					break;
 				}
 			}
 		}
-		bone.getStructure().setRelevantConstraints(constraintList);
+		bone.getStructure().setRelevantConstraints(Functional.toList(Functional.map(constraintList, new IFunction<Constraint, IConstraint>() {
+
+			@Override
+			public IConstraint invoke(Constraint t) {
+				return t;
+			}
+
+		})));
 	}
```

Comment:
Can you please explain this change?

Reply:
Not really. We're all pretty sure we didn't change these lines.

Reply:
https://github.com/FeatureIDE/FeatureIDE/commit/a093743417bb33da260566d7cfa402736d1f66bd

Seems like your branch is behind the develop branch of FeatureIDE.